pkg/monitortests/authentication/requiredsccmonitortests/monitortest.go

```diff
@@ -61,27 +112,70 @@ func (w *requiredSCCAnnotationChecker) CollectData(ctx context.Context, storageD
 
 	junits := []*junitapi.JUnitTestCase{}
 	for _, ns := range namespaces.Items {
-		// require that all workloads in openshift, kube-* or default namespaces must have the required-scc annotation
+		// skip managed service namespaces
+		if exutil.ManagedServiceNamespaces.Has(ns.Name) {
+			continue
+		}
```

Comment:
@stbenjam we have decided to skip scc pinning for the namespaces in `ManagedServiceNamespaces`. If you believe these namespaces should instead be treated as flaking, please let us know.

Reply:
FYI ^^^ Flaking means that the test might sometimes pass and sometimes fail. By treating these namespaces as flaking, we allow our tests to continue without blocking progress. So if we decide to treat them as flaking, those namespaces should be fixed and eventually removed from the flaking list.

Reply:
Skipping them entirely sounds fine, thanks